docs/usage.md

```diff
@@ -1,64 +1,59 @@
 ```
 
-### Full samplesheet
-
-The pipeline will auto-detect whether a sample is single- or paired-end using the information provided in the samplesheet. The samplesheet can have as many columns as you desire, however, there is a strict requirement for the first 3 columns to match those defined in the table below.
-
-A final samplesheet file consisting of both single- and paired-end data may look something like the one below. This is for 6 samples, where `TREATMENT_REP3` has been sequenced twice.
+You may want to skip the alignment step and start from pre-aligned BAM files. In this case your `samplesheet.csv` will look like: 
 
 ```console
-sample,fastq_1,fastq_2
-CONTROL_REP1,AEG588A1_S1_L002_R1_001.fastq.gz,AEG588A1_S1_L002_R2_001.fastq.gz
-CONTROL_REP2,AEG588A2_S2_L002_R1_001.fastq.gz,AEG588A2_S2_L002_R2_001.fastq.gz
-CONTROL_REP3,AEG588A3_S3_L002_R1_001.fastq.gz,AEG588A3_S3_L002_R2_001.fastq.gz
-TREATMENT_REP1,AEG588A4_S4_L003_R1_001.fastq.gz,
-TREATMENT_REP2,AEG588A5_S5_L003_R1_001.fastq.gz,
-TREATMENT_REP3,AEG588A6_S6_L003_R1_001.fastq.gz,
-TREATMENT_REP3,AEG588A6_S6_L004_R1_001.fastq.gz,
+sample,bam,gender
+SAMPLE1,path/to/SAMPLE1.bam,female
+SAMPLE2,path/to/SAMPLE2.bam,male
 ```
 
-| Column    | Description                                                                                                                                                                            |
+| Column    | Description|
 | --------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `sample`  | Custom sample name. This entry will be identical for multiple sequencing libraries/runs from the same sample. Spaces in sample names are automatically converted to underscores (`_`). |
-| `fastq_1` | Full path to FastQ file for Illumina short reads 1. File has to be gzipped and have the extension ".fastq.gz" or ".fq.gz".                                                             |
-| `fastq_2` | Full path to FastQ file for Illumina short reads 2. File has to be gzipped and have the extension ".fastq.gz" or ".fq.gz".                                                             |
+| `fastq_1` | Full path to FastQ file for Illumina short reads 1.                                                         |
+| `fastq_2` | Full path to FastQ file for Illumina short reads 2.     
+| `bam` | Full path to BAM file.                                                          |
+
+| Optional    | Description|
+| --------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `gender` | Gender of the patient that may be used in the copy number analysis; this could be either `female` or `male`. 
+
+> **NB:** You need to use all fastq files **OR** all bam files in the samplesheet.
```

Comment:
Suggestion: "You can either use FASTQ files, or BAM files in a samplesheet. Mixing FASTQ fles and BAM files in the same samplesheet is not supported."